data/trivia/usmap.txt

```diff
@@ -0,0 +1,55 @@
+What USA state of territory is highlighted on this map? https://i.imgur.com/CM2a4cW.png`US Virgin Islands`U.S Virgin Islands`U.S. Virgin Islands`Virgin Islands
+What USA state of territory is highlighted on this map? https://i.imgur.com/h432vHd.png`Puerto Rico
+What USA state of territory is highlighted on this map? https://i.imgur.com/OZcAQ42.png`American Samoa
+What USA state of territory is highlighted on this map? https://i.imgur.com/ig0S0OB.png`Guam
+What USA state of territory is highlighted on this map? https://i.imgur.com/EPdhiIv.png`Northern Mariana Islands`Mariana Islands`
```

Comment:
Please also add Commonwealth of the Northern Mariana Islands or CMNI as it's called here